forumhelpers/FHP.js

```diff
@@ -63,3 +63,37 @@ function getCount(name) {
 			document.getElementById(name+"Posts").innerText = " - "+posts+" Posts";
 		});
 }
+
+
+//Search for a user
+var searchBar = document.getElementById("searchBar")
+var searchButton = document.getElementById("searchButton")
+
+function checkUser() {
+	var searchedUser = searchBar.value.toUpperCase();
+	if (document.getElementById(searchedUser) != undefined) {
+		searchBar.style.borderColor = "green";
+		searchButton.style.borderColor = "green";
+		searchButton.disabled = false;
+		searchButton.title = "Search!";
+	}
+	else if (searchedUser != "") {
+		searchBar.style.borderColor = "red";
+		searchButton.style.borderColor = "red";
+		searchButton.disabled = true;
+		searchButton.title = "Invald Username";
+	}
+	else {
+		searchBar.style.borderColor = "black"
+		searchButton.style.borderColor = "black";
+		searchButton.disabled = true;
+		searchButton.title = "Invald Username";
```

Comment:
*invalid (also on line 84)

Reply:
By the way, is it possible to clear out the tooltip when the user clears the search bar?
<img width="740" alt="Screen Shot 2021-05-15 at 2 16 16 PM" src="https://user-images.githubusercontent.com/82768218/118375581-2112ec00-b588-11eb-8395-dedcf8c89c5c.png">

Reply:
Yes, that is possible. I left the tooltip because the bar being empty would be an invalid username.

Reply:
ig that makes sense